### PR TITLE
Remove a bunch more indirection.

### PR DESCRIPTION
**Pull request by mikepurvis**
Mainly affects the inertia properties.

@Shokoofeh to review.

### CODE REVIEW

**Comment by mikepurvis** [comment]
Shouldn't be any functional change here. I did a diff on the xacro processed output, and it's identical apart from comment changes.

File: grizzly_description/urdf/base.urdf.xacro [hunk 1]
```diff
@@ -172,20 +128,10 @@ This program contains the description of the robot base model for the Clearpath
...
-                 lr="right"
-                 parent="chassis_link"
-                 translateX="-1"
-                 translateY="-1"
-                 translateZ="0"
-                 flipY="1" />
+  <grizzly_wheel fr="rear" lr="left" flipY="0" parent="chassis_link"
+                 translateX="-1" translateY="1" />
+  <grizzly_wheel fr="rear" lr="right" flipY="1" parent="chassis_link"
+                 translateX="-1" translateY="-1" />
```
Comment (shokoofeh) [inline grizzly_description/urdf/base.urdf.xacro]: Only for more readability, I suggest to follow the same order that you used for front wheels: fr  lr  parent  translateX  translateY  flipY

**Comment by shokoofeh** [comment]
I test it in simulation. Everything if fine. LGTM

File: grizzly_description/urdf/base.urdf.xacro [hunk 2]
```diff
@@ -147,16 +103,16 @@ This program contains the description of the robot base model for the Clearpath
     <child link="navsat_link" />
   </joint>
 
+  <!-- The position and orientation of the IMU in the rear compartment. -->
   <link name="imu_link" />
   <joint name="imu_joint" type="fixed">
     <origin xyz="-${chassis_length/2} 0 0"
-            rpy="0 0 0" />
+            rpy="${M_PI} ${M_PI/2} 0" />
```
Comment (mikepurvis) [inline grizzly_description/urdf/base.urdf.xacro:110]: This orientation should have remained, as the IMU in the user bay is not mounted level with the robot— sorry for the confusion here. This shouldn't impact robot_localization, I don't think, should it?

Comment (shokoofeh) [inline grizzly_description/urdf/base.urdf.xacro:110]: This orientation does impact robot_localization. 
As we discussed, the orientation will be set as it is mount in the user bay but in the imu controller plugin for Gazebo the frame id will be set as base_link.

**Pull request merged**